main.js

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable linebreak-style */
+const textAreaAttributes = {
+    name: 'textarea',
+    id: 'textarea',
+    cols: '25',
+    rows: '15',
+    placeholder: 'Type your text here (made for Windows)',
+};
+
+function createTextArea() {
+    const textArea = document.createElement('textarea');
+    Object.entries(textAreaAttributes).forEach((element) => {
+        textArea.setAttribute(element[0], element[1]);
+    });
+    return textArea;
+}
+
+let Language = localStorage.value || 'en';
```

Comment:
It is not usual case to use Upper case for variable name.
Use upper case for classes.